SU-SWS/saa_victoria_subtheme
# Adapt 719  sidebar nav external links

# READY FOR REVIEW

# Summary
- Add support for black external link icon in sidebar navigation (external links module adds that class automatically)

# Review By (Date)
- TODAY 9/18

# Urgency
- medium

# Steps to Test

1. Pull down the branch or test on https://saauserguide.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/cards/square-vertical-11-card  and see the external link in the sidebar.
2. Make sure the icon is black and then cardinal red on hover, matching the link text.

# Affected Projects or Products
- N/A

# Associated Issues and/or People
- JIRA ticket ADAPT-719
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
